Run3Detector/analysis/utilities/milliqanCuts.py

```diff
@@ -6,6 +6,30 @@ class milliqanCuts():
 
     def __init__(self):
         self.events = []
+        self.cutflow = []
+        self.counter = 0
+
+    def cutflowCounter(self):
+        # Increments events passing each stage of the cutflow
+        # Creates each stage during the first pass
+        try:
```

Comment:
could this just be an if else?

`if self.counter < len(self.cutflow): 
    ...
else:
    self.cutflow.append`